src/HeaderCell.tsx

```diff
@@ -89,13 +87,32 @@ export default function HeaderCell<R, SR>({
     window.addEventListener('pointermove', onPointerMove);
     window.addEventListener('pointerup', onPointerUp);
   }
+  const sortDirection = sortColumns?.find(item => item.columnKey === column.key)?.direction;
+  const index = sortColumns?.findIndex(sort => sort.columnKey === column.key);
+  const priority = index !== undefined ? index + 1 : undefined;
+
+  const onSort = (columnKey: string, direction: SortDirection, ctrlClick: boolean) => {
```

Comment:
I have seen a few grids using `shift` instead of `ctrl` for multi-sorting. I wonder if we should use a generic parameter like `isMulti` instead of `ctrlClick` and then it can be customized using `headerRender` as needed